### PR TITLE
Improve the Content-Security-Policy (CSP) readme with content

### PR DESCRIPTION
**Pull request by shaialon**
- Explain more on client-side attacks.
- Add reference to CSP scanner.

### CODE REVIEW

File: middlewares/content-security-policy/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Content Security Policy middleware
 
-Content Security Policy helps prevent unwanted content being injected into your webpages. This can mitigate cross-site scripting (XSS) vulnerabilities, malicious frames, unwanted trackers, and much more.
+Content Security Policy (CSP) helps prevent unwanted content from being injected/loaded into your webpages. This can mitigate cross-site scripting (XSS) vulnerabilities, Clickjacking, formajacking, malicious frames, unwanted trackers, client-side injected malware, and other [web client-side attacks](https://cspscanner.com/client-side-attacks).
```
Comment (EvanHahn) [inline middlewares/content-security-policy/README.md:3]: 1. Could you make "clickjacking" lowercase?
1. Should "formajacking" be "formjacking"?
1. Could you remove "client-side injected malware"? It feels redundant with the last clause.
1. Could you remove the link?

File: middlewares/content-security-policy/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ You can set any directives you wish. `defaultSrc` is required, but can be explic
 
 The `reportOnly` option, if set to `true`, sets the `Content-Security-Policy-Report-Only` header instead.
 
-This middleware does minimal validation. You should use a more sophisticated CSP validator, like [Google's CSP Evaluator](https://csp-evaluator.withgoogle.com/), to make sure your CSP looks good.
+This middleware does minimal validation. You should use a more sophisticated CSP validator, like [Google's CSP Evaluator](https://csp-evaluator.withgoogle.com/), or [CSP Scanner](https://cspscanner.com/) to make sure your CSP is both valid, and effective at mitigating attacks.
```
Comment (EvanHahn) [inline middlewares/content-security-policy/README.md:33]: Could you revert this change?

File: middlewares/content-security-policy/README.md [hunk 3]
```diff
@@ -61,5 +61,6 @@ app.use((req, res) => {
 ## See also
 
 - [Google's CSP Evaluator tool](https://csp-evaluator.withgoogle.com/)
+- [CSP Scanner - Validate and Improve a Content Security Policy](https://cspscanner.com/)
```
Comment (EvanHahn) [inline middlewares/content-security-policy/README.md:64]: Could you just change this link to "CSP Scanner"?

**Review by EvanHahn — CHANGES_REQUESTED** [review]
Thanks for doing this! Left a few comments.

**Comment by shaialon** [comment]
Thanks @EvanHahn . Done 🚀

**Review by EvanHahn — APPROVED** [review]

**Comment by EvanHahn** [comment]
Thanks!

**Comment by shaialon** [comment]
@EvanHahn - I see this was not deployed:
https://www.npmjs.com/package/helmet-csp

I see it also has different versions, so wonder how it's setup.

**Comment by EvanHahn** [comment]
Thanks for pointing this out.

Unless there's something very substantial, I don't usually deploy new versions if documentation is the only thing that's changed. The next time helmet-csp updates, I'll be sure to push these changes out.

Hope that answers your question!

**Pull request merged**